src/scss/base.scss

```diff
@@ -1,15 +1,16 @@
+
 @include oFontsInclude(BentonSans, normal);
 
-.o-ft-header {
+.o-header {
 	// 80 px for the primary container and 5 for an empty secondary or 25 for a secondary with content
 	min-height: 85px;
 	max-height: 125px;
 	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
```

Comment:
That's a great point, I've thought about this recently too. Until now it hasn't been a problem so I'm not sure we should change it yet.